react/components/AutocompleteInput/SearchInput/index.tsx

```diff
@@ -0,0 +1,118 @@
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+
+import IconSearch from '../../icon/Search'
+
+import ClearInputIcon from '../../icon/Clear'
+
+const propTypes = {
+  /** Determine if the input's bottom corners should be rounded or not */
+  roundedBottom: PropTypes.bool,
+  /* Input props */
+
```

Comment:
Missing prop?

Reply:
Actually, it's just to comment that these props are related to the input element itself. I'll make it closer to the `value` prop documentation.